docs/topics/publishing.md

```diff
@@ -152,3 +162,9 @@ Execute the commands in your local checked out repositories and push the tags us
 ```
 git push --tags
 ```
+
+## Announce new versions
```

Comment:
Aren't we considering doing the per-announcements about the upcoming security releases, similar to the ones WMF does for mediawiki security releases?
The idea of these, as I get it, is to give maintainers a heads up that there will be security release coming up in 24 hours or so, so that they can plan for a timely update (vs being informed "there has been security fix published and the vulnerability is now publicly know"). As we consider serious security problems, timeliness might be potentially of relevance here?